### ISSUE TITLE
Clues should not be normalized to lowercase

### ISSUE THREAD

**Issue by wctaiwan** [an-empty-string/codenames#10]
The bot turns all clues into lowercase (e.g. `Jaws` becomes `jaws`). Instead case should be preserved.
